packages/syntax-core/src/Badge/Badge.tsx

```diff
@@ -48,13 +55,19 @@ const Badge = ({
     paddingY={1}
     rounding="full"
     backgroundColor={color}
+    dangerouslySetInlineStyle={{ __style: { lineHeight: "14px" } }}
```

Comment:
👀 👀 The design calls for badge of always 22px height. 4px top + bottom padding, 14px line-height, 12px font-size, with a 16px w/h Icon size.

Typography does not set line-height for any given size (I think it should!), so I am hard-coding the 14px line-height here.

Otherwise it is calculated to 15px, which makes the overall height of the Badge 23px instead of 22px